java/basic-async-bot/src/main/java/com/google/chat/bot/basic/async/Bot.java

```diff
@@ -0,0 +1,106 @@
+/**
+ * Copyright 2017 Google Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.google.chat.bot.basic.async;
+
+// [START async-bot]
```

Comment:
hangouts-async-bot for tracking?

Reply:
yes, though prefer to defer retagging to a separate PR (have to coordinate change w/updating docs)